Source/UI/NewUI/BillingForm/View/Field/BillingFormTextFieldView.swift

```diff
@@ -0,0 +1,171 @@
+import UIKit
+
+protocol BillingFormTextFieldViewDelegate: AnyObject {
+    func textFieldShouldBeginEditing(textField: UITextField)
+    func textFieldShouldReturn()
+    func textFieldDidEndEditing(textField: UITextField)
+    func textFieldDidChangeCharacters(textField: UITextField, replacementString: String)
+}
+
+final class BillingFormTextFieldView: UIView {
+    
+    private var style: BillingFormTextFieldCellStyle
+    private var type: BillingFormCellType
+    private weak var delegate: BillingFormTextFieldViewDelegate?
+    
+    private(set) lazy var headerLabel: UILabel = {
+        let view = UILabel()
+        view.text = style.title?.text
+        view.font = style.title?.font
+        view.textColor = style.title?.textColor
+        view.numberOfLines = 0
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        return view
+    }()
+    
+    private(set) lazy var hintLabel: UILabel = {
+        let view = UILabel()
+        view.text = style.hint?.text
+        view.font = style.hint?.font
+        view.textColor = style.hint?.textColor
+        view.numberOfLines = 0
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        return view
+    }()
+    
+    private(set) lazy var textFieldContainer: UIView = {
```

Comment:
Why are the UI components here private(set)? Tried searching for some but can't find them being used externally, so private seemed appropriate?

Reply:
for testing purpose, 
 XCTAssertEqual(view.textFieldContainer.layer.borderColor, style.textfield.normalBorderColor.cgColor)